lib/Controller/LocalController.php

```diff
@@ -628,6 +632,15 @@ public function documentsCache(array $documents): DataResponse {
 	 * @throws AccountDoesNotExistException
 	 */
 	private function initViewer(bool $exception = false) {
+		// only initialize viewer data for logged in users
+		if ($this->userId === null) {
```

Comment:
@daita Maybe you can help here. Is there a good way to still get details for actors if no viewerId is set?

Reply:
what about using LocalController::accountInfo() and add false as a parameter of `initViewer`

```
			$this->initViewer(false);
```

Reply:
Hm let me check that again, since we actually don't need the following/follower (if the current user is following) details, just the number of post/followers/following.